### PR TITLE
[Bugfix] Support file destination and output

### PR DESCRIPTION
**Pull request by rldhont**
file/html output paths should be added in the input commands of the building R script.

### CODE REVIEW

File: processing_r/processing/algorithm.py [hunk 1]
```diff
@@ -645,7 +645,13 @@ def build_import_commands(self,  # pylint: disable=too-many-locals,too-many-bran
                     layer_idx += 1
                 s += ')'
                 commands.append(s)
-            # TODO folder, file/html output paths should be set here
+
+        # TODO folder, file/html output paths should be set here
```
Comment (nyalldawson) [inline processing_r/processing/algorithm.py:649]: This line can be revised now, right?

Comment (rldhont) [reply processing_r/processing/algorithm.py]: Add folder destination path in input commands.

**Comment by nyalldawson** [comment]
Oh sorry - I keep forgetting, can you update the docs too?

**Comment by rldhont** [comment]
Docs updated too

**Comment by nyalldawson** [comment]
Thank you! I'll push out a new version today

**Pull request merged**